src/run.ts

```diff
@@ -142,17 +138,16 @@ export async function runPublish({
     }
   }
 
+  const result: PublishResult = { published: true, publishedPackages: [] };
```

Comment:
I thought a lot about this. This somewhat changes the meaning of `published`, with this, it means: as long as the publishing script runs fine, we consider `published=true`, in a way, this is coherent with the definition of the flag in our README file: `A boolean value to indicate whether a publishing is happened or not`. Like I said in the issue created, I think the main issue here is we need a flag to tell us in the Github action script that whether `publishing` or `versioning (creating the New Version PR)` happened.

Reply:
I agree that relying on the `New tag:` output is less than ideal. However, you can (sort of) check today which script has been executed by inspecting the `hasChangesets` output:
https://github.com/changesets/action/pull/136/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R165

Reply:
hm, thinking about it, looks like my approach is quite wrong though. Will close this PR.